### PR TITLE
ci/cd and auth sections updates

### PR DESCRIPTION
**Pull request by Fomich3000**

### CODE REVIEW

File: docs/8base-console/platform-tools/ci-cd.md [hunk 1]
```diff
@@ -61,11 +171,33 @@ When changes have been made to your current workspace environment and you're rea
 #   --environment, -e Target environment
 ```
 
-Once successfully run, a timestamped migrations script will appear in a directory called `migrations` that's in the root of your project.
+It generates new migration files in your local `migrations` project directory.
+
+These are automatically generated by the server after changes are made in a set Environment (for example - a new table is added). Developers can also manually create their own migration files and commit them directly to specified environments.
+
+Generating migrations for data in the User's tables occurs differently. For User data, you must specify a *table name* flag in command argument. For example:
+`migration generate -t <tableName>` Note that this kind of migration returns the whole state of the requested table's records without any comparison to the target Environment.
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md:179]: @Fomich3000
do we have issues with migrations of large data?

File: src/data/doc-links.yaml [hunk 2]
```diff
@@ -124,8 +124,6 @@
           link: /docs/development-tools/cli/commands
         - title: Generators
           link: /docs/development-tools/cli/generators
-        - title: CI/CD
-          link: /docs/development-tools/cli/ci-cd
```
Comment (Selnapenek) [inline src/data/doc-links.yaml:128]: do we have a mention of this link?

File: docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -160,14 +160,14 @@ When using the 8base CLI you need to authenticate your development workspace. Th
 ### 3.2 Deploy a Serverless Function (**OPTIONAL**)
 This step is **totally optional** in the quickstart. It will show you how to quickly extend your workspace by generating and deploying serverless functions to a workspace. 
 
-Move into the `8base-starter-app/server` directory. Let's initialize a new server project and configure it to a hosted workspace. Using the `init` and `configure` commands will generate and configure the needed resources. The workspace you select is the one where the Custom Functions will be deployed to when ready.
+Move into the `8base-starter-app` directory. Let's initialize a new server project and configure it to a hosted workspace. Using the `init` and `configure` commands will generate and configure the needed resources. The workspace you select is the one where the Custom Functions will be deployed to when ready.
```
Comment (Selnapenek) [inline docs/getting-started/quick-start.md:163]: does the `configure` command works in this case?

File: docs/getting-started/quick-start.md [hunk 4]
```diff
@@ -160,14 +160,14 @@ When using the 8base CLI you need to authenticate your development workspace. Th
...
 
 ```text
-# Change into the server directory
-cd ../server
+# Init new 8base project in server folder with one function called myCustomResolver
+8base init server --functions=resolver:myCustomResolver
```
Comment (Selnapenek) [inline docs/getting-started/quick-start.md:167]: does this work correct if we already have `server` dir?

File: docs/8base-console/platform-tools/ci-cd.md [hunk 5]
```diff
@@ -1,52 +1,162 @@
-# Continuous Integration & Continuous Deployment (CI/CD)
+# CI/CD System
 
-8base built CI/CD capabilities natively into the workspace to help developers and teams easily management professional software quality controls when developing their applications. On subscription plans where CI/CD is enabled, developers will be able to create up to a specified number of enviroments and graduate changes through their environment pipeline in a [Git](https://git-scm.com/) like fashion.
+Teams that have multiple developers working simultaneously on different features of the same application – or [Workspace](/docs/getting-started#workspaces) – benefit from having isolated development environments for each developer. 8base built CI/CD capabilities natively into the workspace to make this possible!
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md:3]: maybe better?
```suggestion
Teams that have multiple developers working at the same time on different features of the one application – or [Workspace](/docs/getting-started#workspaces) – benefit from having isolated development environments for each developer. 8base built CI/CD capabilities natively into the workspace to make this possible!
```

File: docs/8base-console/platform-tools/ci-cd.md [hunk 6]
```diff
@@ -1,52 +1,162 @@
...
 
-## Getting Started with CI/CD
+ On subscription plans where CI/CD is enabled, developers will be able to create up to a specified number of enviroments and graduate changes through their environment pipeline in a [Git](https://git-scm.com/) like fashion.
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md]: think we need update this page to https://www.8base.com/pricing

Comment (Selnapenek) [reply docs/8base-console/platform-tools/ci-cd.md]: what is the limit of envs?

Comment (Selnapenek) [reply docs/8base-console/platform-tools/ci-cd.md]:
```suggestion
 On subscription plans where CI/CD is enabled, developers will be able to create up to a specified number of environments and graduate changes through their environment pipeline in a [Git](https://git-scm.com/) like fashion.
```

File: docs/8base-console/platform-tools/ci-cd.md [hunk 7]
```diff
@@ -1,52 +1,162 @@
...
+
+### Environment rules
+
+1. Environments are inheritable.
+2. You CANNOT manually change *System Parts* of parent Environments after Branching (for example, one Environment can’t be manually changed after another is created from it).
+3. You CAN manually change *System Parts* and deploy in child Environments (inheritors).
+4. Every Environment can have up to 3-inheritors.
 
-When creating new *Environments*, you are able to specify whether you just want to copy the system configuration into the new environment (Tables, Functions, Roles, etc.) or whether you want to copy the system configuration and all data records.
+Changing/deploying to parent Environments is only possible with [Migration logic](/docs/development-tools/cli/ci-cd#migrations-logic-and-commands).
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md:32]: `/docs/development-tools/cli/ci-cd` you remove this link?

File: docs/8base-console/platform-tools/ci-cd.md [hunk 8]
```diff
@@ -1,52 +1,162 @@
...
+
+You can read more about [System Parts](/docs/getting-started#system-parts) and [User Parts](/docs/getting-started#user-parts) of 8base in our [overview section](/docs/getting-started#8base-application-structure).
+
+## CI/CD Commands
+
+The CI/CD System is controlled using the [8base CLI](/docs/development-tools/cli). There are 3 command categories that correspond to the main components of the CI/CD System; *Environment commands*, *Migrations commands*, and *Backup commands*.
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md:38]: maybe better save this page `/docs/development-tools/cli/ci-cd` ?

Comment (Selnapenek) [reply docs/8base-console/platform-tools/ci-cd.md:38]: I mean at `cli/ci-cd` we describe only descriptions of commands (like man or help).
and on this page, we describe the use case of how to use cli for migrations.
What do you think?

File: docs/8base-console/platform-tools/ci-cd.md [hunk 9]
```diff
@@ -78,24 +210,87 @@ Once a migration has been generated, you'll be able to commit it so that the cha
...
+
+## Basic CI/CD Workflow
+
+A basic CI/CD workflow can resemble something like this:
+
+### Environments
+
+- `Master` - gets used as the "Production" environment (live application backend).
+- `Staging` environment gets branched from `Master` (pre go-live backend for testing).
+- `Dev` environment gets branched from `Staging` (development backend for whole team).
```
Comment (Selnapenek) [inline docs/8base-console/platform-tools/ci-cd.md:284]: lower case it is better for URL
`api.8base.com/123id_dev`

File: docs/getting-started/quick-start.md (hunk 3 above)
Comment (Fomich3000) [reply docs/getting-started/quick-start.md:163]: Added more details in new commit

File: docs/8base-console/platform-tools/ci-cd.md (hunk 5 above)
Comment (Fomich3000) [reply docs/8base-console/platform-tools/ci-cd.md:3]: I guess it has been already reviewed by Sebastian in the past PR.

**Pull request closed without merging**